nexus-api/src/routes/v0/bootstrap/user.rs

```diff
@@ -0,0 +1,40 @@
+use crate::Result;
+use crate::{routes::v0::endpoints::USER_BOOTSTRAP_ROUTE, Error};
+use axum::extract::Path;
+use axum::Json;
+use nexus_common::models::bootstrap::{Bootstrap, ViewType};
+use tracing::info;
+use utoipa::OpenApi;
+
+#[utoipa::path(
+    get,
+    path = USER_BOOTSTRAP_ROUTE,
+    description = "Initial payload for all data required to bootstrap the pubky.app application. The client app will request it while the user is performing sign-in in order to pre-populate the client DB",
+    tag = "User",
```

Comment:
```suggestion
    tag = "Bootstrap",
```
It's best if it is its own category on swagger-ui